app/services/sandbox_url_updator.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
```

Comment:
This needs some tests, with different example URLs that should vs should not validate.

I tested it out locally, and it throws an invalid URL error when I didn't expect it to, for an English Wikipedia Assignment trying to updated it to the URL "https://en.wikipedia.org/wiki/User:Sage_(Wiki_Ed)/test"

Reply:
Now i have fixed the regex pattern matching as previously it was not expecting semicolon after wiki/  . Adding more tests like with spaces, without spaces, with special characters etc in url.